src/core/tests/Test_StatusHandlerTruncation.py

```diff
@@ -0,0 +1,784 @@
+        self.assertEqual(assessment_msg['otherPatchCount'], assessment_truncated_msg['otherPatchCount'])
+        self.assertEqual(assessment_msg['startTime'], assessment_truncated_msg['startTime'])
+        self.assertEqual(assessment_msg['lastModifiedTime'], assessment_truncated_msg['lastModifiedTime'])
+        self.assertEqual(assessment_msg['startedBy'], assessment_truncated_msg['startedBy'])
+
+    def __assert_installation_truncated_msg_fields(self, complete_substatus_file_data, truncated_substatus_file_data, substatus_index=0):
+        installation_msg = self.__get_message_json_from_substatus(complete_substatus_file_data, substatus_index=substatus_index)
+        installation_truncated_msg = self.__get_message_json_from_substatus(truncated_substatus_file_data, substatus_index=substatus_index)
+
+        self.assertEqual(installation_msg['installationActivityId'], installation_truncated_msg['installationActivityId'])
+        self.assertEqual(installation_msg['rebootStatus'], installation_truncated_msg['rebootStatus'])
+        self.assertEqual(installation_msg['maintenanceWindowExceeded'], installation_truncated_msg['maintenanceWindowExceeded'])
+        self.assertEqual(installation_msg['notSelectedPatchCount'], installation_truncated_msg['notSelectedPatchCount'])
+        self.assertEqual(installation_msg['excludedPatchCount'], installation_truncated_msg['excludedPatchCount'])
+        self.assertEqual(installation_msg['pendingPatchCount'], installation_truncated_msg['pendingPatchCount'])
+        self.assertEqual(installation_msg['installedPatchCount'], installation_truncated_msg['installedPatchCount'])
+        self.assertEqual(installation_msg['failedPatchCount'], installation_truncated_msg['failedPatchCount'])
+        self.assertEqual(installation_msg['startTime'], installation_truncated_msg['startTime'])
+        self.assertEqual(installation_msg['lastModifiedTime'], installation_truncated_msg['lastModifiedTime'])
+        self.assertEqual(installation_msg['maintenanceRunId'], installation_truncated_msg['maintenanceRunId'])
+
+    def __get_message_json_from_substatus(self, substatus_file_data, substatus_index=0):
+        return json.loads(substatus_file_data["status"]["substatus"][substatus_index]["formattedMessage"]["message"])
+
+    def __convert_performance_time_to_date_time_format(self, performance_time):
+        performance_time = abs(performance_time)
+
+        # Calc days, hours, minutes, and seconds
+        days, remainder = divmod(performance_time, 86400)  # 86400 seconds in a day
+        hours, remainder = divmod(remainder, 3600)  # 3600 seconds in an hour
+        minutes, seconds = divmod(remainder, 60)  # 60 seconds in a minute
+
+        # Format the result
+        formatted_time = "%d days, %d hours, %d minutes, %.6f seconds" % (int(days), int(hours), int(minutes), seconds)
+        return formatted_time
+
+    def __create_tmp_file_for_log_and_set_stdout(self):
```

Comment:
Should not be in use after recent changes

Reply:
logic is removed

done.